test/main_test.rb

```diff
@@ -1,5 +1,4 @@
-require "test_helper"
-require "./main.rb"
+require 'main'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.